src/main/scala/com/github/unchama/seichiassist/subsystems/gachapoint/domain/gachapoint/GachaPoint.scala

```diff
@@ -16,7 +16,7 @@ case class GachaPoint(exp: SeichiExpAmount) {
     (exp.amount /% GachaPoint.perGachaTicket.exp.amount)._1.toBigInt
 
   /**
-   * ガチャポイントをバッチでガチャ券に変換した際のポイントの変化を計算する。
+   * ガチャポイントを576個(= 64 * 9スタック)のバッチでガチャ券に変換した際のポイントの変化を計算する。
    */
   lazy val useInBatch: GachaPoint.Usage = {
```

Comment:
`useInBatch` が576個のバッチを扱うことを変数名としても明確にしたいです。

```suggestion
  lazy val useInLargeBatch: GachaPoint.Usage = {
```